apps/judicial-system/backend/src/app/modules/case/dto/internalCreateCase.dto.ts

```diff
@@ -38,6 +38,14 @@ export class InternalCreateCaseDto {
   @ApiProperty({ type: String })
   readonly prosecutorNationalId!: string
 
+  @IsOptional()
+  @IsString()
+  @MinLength(10)
```

Comment:
Nit: Consider using `Length` instead for a bit cleaner code

<img width="850" alt="Screenshot 2025-03-19 at 13 06 20" src="https://github.com/user-attachments/assets/73dbf0d7-c787-4d4b-aa5c-37075faefcdb" />

Reply:
Thanks, I wasn't aware of this class validator.